src/main/java/jimmy/mcgymmy/model/History.java

```diff
@@ -34,7 +34,7 @@ boolean empty() {
 
     /**
      * Get the previous state from history
-     * @throws EmptyStackException
+     * @throws AssertionError
```

Comment:
I think we need to change this to EmptyStackException too.

Reply:
done!